src/gtk/menu.cpp

```diff
@@ -1110,6 +1139,20 @@ void wxMenu::Attach(wxMenuBarBase *menubar)
     SetLayoutDirection(menubar->GetLayoutDirection());
 }
 
+void wxMenu::SetupBitmaps(wxWindow *win)
+{
+    wxMenuItemList::compatibility_iterator node = GetMenuItems().GetFirst();
+    while (node)
+    {
+        wxMenuItem *menuitem = node->GetData();
+        if (menuitem->IsSubMenu())
+            menuitem->GetSubMenu()->SetupBitmaps(win);
+        if (menuitem->GetKind() == wxITEM_NORMAL)
```

Comment:
Do we really need to test for this? AFAICS we could just always call `SetupBitmaps()` and it would do nothing for the others.

It doesn't change anything right now, of course, but if we ever allow bitmaps for check/radio items or add some new menu item kind, it would prevent the bitmaps from being used for it and it could be not obvious to find why.